jetstream/allocation.py

```diff
@@ -122,6 +122,11 @@ def report_project_allocation(self, report_id, username, project_name, su_total,
         if not type(su_total) in [int, float]:
             raise Exception("SU total should be integer or float")
 
+        # Abort here to prevent reporting of usage, should abort in all
+        # development environments to prevent double reporting
+        if not settings.REPORT_ALLOCATION_USAGE:
+            return True
```

Comment:
I think this should probably return False, not True. If this returns True the calling code will mark the report as sent.

Reply:
And/or use this flag higher up in the call tree.

Reply:
I wanted it to mark the report as sent, so that the app wouldn't continue to accumulate reports. Essentially this flag will convince atmo that the reports were received. Its sort of akin to mocking the tas api to return a 200 on sending them. However I could see it leaving someone puzzled that the db flags them as sent, but the reports were never actually sent.

I'll move it up so the flag makes the `report_allocations_to_tas` a no-op.